docs/user-guide/labels.md

```diff
@@ -94,23 +112,25 @@ tier != frontend
 ```
 
 The former selects all resources with key equal to `environment` and value equal to `production`.
-The latter selects all resources with key equal to `tier` and value distinct from `frontend`.
+The latter selects all resources with key equal to `tier` and value distinct from `frontend` and all resources with no labels with the `tier` key.
```

Comment:
add a comma before "and" even though it is grammatically incorrect, to make it easier to parse this sentence